### PR TITLE
Adding big commands as sbt tasks and test hook to require big services to be up prior to sbt test execution.

### PR DESCRIPTION
**Pull request by jwinder**

### CODE REVIEW

**Comment by jwinder** [comment]
This is missing a few cases. It doesn't run `big up` during `test-only`. It might be a good idea to add it to the `run` task as well.

**Comment by zcox** [comment]
This is looking really nice.

**Comment by rubbish** [comment]
me like. :+1: 

@jwinder Anything more you want to do to this?

**Comment by adamdecaf** [comment]
Is it possible to add a test?

**Comment by jwinder** [comment]
I think it's good to go. I considered have a `big` task that accepts arguments so that arbitrary big commands can be run. Does that sound like a good idea?

I'll look into writing a test.

File: src/main/scala/BannoBig.scala [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+  val services = SettingKey[Seq[String]]("big-services")
+
+  val doctor = TaskKey[Unit]("big-doctor")
+  val up = TaskKey[Unit]("big-up")
+  val upServices = TaskKey[Unit]("big-up-services")
+  val upServicesIfAutocreate = TaskKey[Unit]("big-up-services-if-autocreate")
+  val ps = TaskKey[Unit]("big-ps")
+  val kill = TaskKey[Unit]("big-kill")
+  val killServices = TaskKey[Unit]("big-kill-services")
```
Comment (DustinChaloupka) [inline src/main/scala/BannoBig.scala]: Do we want to do a `big destroy` instead or is `big kill` good enough?  The `destroy` just does a `kill` then `rm` on the images.

**Comment by DustinChaloupka** [comment]
:+1: This is great.  Didn't even know there was a `--no-recreate` flag.

Having the `big` task for arbitrary big commands might be overkill since everything I think that would be used is already here, but it may be nice still.

**Comment by jwinder** [comment]
This should be good to go. @rubbish @DustinChaloupka

**Comment by adamdecaf** [comment]
:+1:

**Comment by DustinChaloupka** [comment]
:100:

**Comment by jwinder** [comment]
I knew that would fail. Will big be available in our new jenkins images?

**Comment by jwinder** [comment]
:x: Blocked on jenkins slaves as mesos tasks for now.

**Comment by rubbish** [comment]
I switch over the jenkins that this PR job is using.

**Comment by jwinder** [comment]
retest this please

**Comment by rubbish** [comment]
It passed for me locally: `[success] Total time: 178 s, completed May 21, 2015 8:38:47 AM`

**Comment by jwinder** [comment]
Is the new jenkins up and running yet? I didn't know if we wanted to wait for that.

**Comment by rubbish** [comment]
It is. I can try the job on there too.

**Pull request merged**